documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -214,11 +218,12 @@ spec:
 EOF
 ----
 <1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
-<2> Specify the VMware data storage MOR, the {rhv-short} storage domain UUID, or the {osp} `volume_type` UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+<2> Specify the VMware data storage MOR, the {rhv-short} storage domain UUID, or the {osp} `volume_type` UUID. For OVA, use `name: Dummy storage for source provider provider-ova` instead of `id: <source_datastore>`.
```

Comment:
I think we should change the `provider-ova` here also to `<provider-name> otherwise its confusing a bit. (in addition to the note later on).